lib/project_types/script/messages/messages.rb

```diff
@@ -45,9 +45,9 @@ module Messages
 
           app_not_installed_cause: "App not installed on store.",
 
-          app_script_not_pushed_help: "Push the script and then try this command again.",
+          app_script_not_pushed_help: "Script isn't on the app. Run {{command:shopify push}}, and then try again.",
 
-          app_script_undefined_help: "Push script to app.",
+          app_script_undefined_help: "Script isn't on the app. Run {{command:shopify create}}, and then try again.",
```

Comment:
did you mean `shopify push` here?
```suggestion
          app_script_undefined_help: "Script isn't on the app. Run {{command:shopify push}}, and then try again.",
```

Reply:
yes